src/types/triggers.ts

```diff
@@ -24,35 +24,78 @@ export enum TriggerType {
   FIRST_FAILURE_VERSION_NAME = "first-failure-in-version-with-name",
 }
 
+export enum TaskTriggers {
+  TASK_STARTS = "task-starts",
+  TASK_FINISHES = "task-finishes",
+  TASK_FAILS = "task-fails",
+  TASK_FAILS_OR_BLOCKED = "task-fails-or-blocked",
+  TASK_SUCCEEDS = "task-succeeds",
+  TASK_EXCEEDS_DURATION = "task-exceeds-duration",
+  TASK_RUNTIME_CHANGE = "task-runtime-change",
+}
+
+export enum VersionTriggers {
+  VERSION_FINISHES = "version-finishes",
+  VERSION_FAILS = "version-fails",
+  VERSION_SUCCEEDS = "version-succeeds",
+  VERSION_EXCEEDS_DURATION = "version-exceeds-duration",
+  VERSION_RUNTIME_CHANGE = "version-runtime-change",
+  BUILD_VARIANT_FINISHES = "build-variant-finishes",
+  BUILD_VARIANT_FAILS = "build-variant-fails",
+  BUILD_VARIANT_SUCCEEDS = "build-variant-succeeds",
+}
+
+export enum ProjectTriggers {
```

Comment:
this may be an old bug, but these two should not be on the project page
<img width="408" alt="image" src="https://user-images.githubusercontent.com/13104717/175553042-5bd4fd14-2ba8-4e16-bb55-67fab4a43940.png">